lib/procore/response.rb

```diff
@@ -72,7 +73,7 @@ def parse_body
 
     def parse_pagination
       headers[:link].to_s.split(", ").map(&:strip).reduce({}) do |links, link|
-        url, name = link.match(/vapid\/(.*?)>; rel="(\w+)"/).captures
+        url, name = link.match(/#{api_version}\/(.*?)>; rel="(\w+)"/).captures
```

Comment:
anyone know why this fails the test?  running this regex in the console works as expected....